modules/jupyter/jupyter_config/config-selfauth-autopilot.yaml

```diff
@@ -111,9 +112,17 @@ singleuser:
           volumeAttributes:
             bucketName: ${gcs_bucket}
             mountOptions: "implicit-dirs,uid=1000,gid=100"
+      - name: secret-volume
+        secret:
+          secretName: ${secret_name}
+          optional: true
+
     extraVolumeMounts:
         - name: data-vol
-          mountPath: /persist-data
+          mountPath: /data
```

Comment:
we don't need persist-data anymore? awesome! this always confused me.

Reply:
Glad you agree.